Sources/Sentry/SentryProfiler.mm

```diff
@@ -622,6 +649,11 @@ - (void)captureEnvelope
     }
     profile[@"transactions"] = transactionsInfo;
 
+    if ([((NSArray *)profile[@"profile"][@"samples"]) count] == 0) {
```

Comment:
Couldn't we just check `[samples count]`? - it's a local var in this scope I believe

Reply:
ooh good find